package.json

```diff
@@ -18,19 +18,20 @@
   "types": "lib/peg.d.ts",
   "bin": "bin/peggy",
   "scripts": {
-    "pretest": "npm run parser",
-    "test": "npm run test:bare",
-    "test:bare": "mocha --recursive test --exclude test/browser.stub.js --reporter min",
+    "clean": "rimraf -f build && rimraf -f browser && mkdir build && cd build && mkdir ts && mkdir rollup && mkdir terser && cd .. && mkdir browser",
+    "terser": "terser -c passes=2 -m --sequences 40 build/rollup/peg.umd.js > build/terser/peggy.umd.min.js && terser -c passes=2 -m --sequences 40 --module build/rollup/peg.esmodule.js > build/terser/peggy.esmodule.min.js",
+    "rollup": "rollup -c",
+    "ts": "tsc --build tsconfig.json",
+    "set_version": "node -e \"const fs = require('fs'), pkg = JSON.parse(fs.readFileSync('./package.json')); fs.writeFileSync('./lib/version.js', '\\n\\'use strict\\';\\n\\nmodule.exports = \\'' + pkg.version + '\\';\\n');\"",
```

Comment:
Put a script in tools for this, please, so we can lint it, etc.

Reply:
okay

Reply:
Done